docs/prysm-usage/p2p-host-ip.md

```diff
@@ -0,0 +1,115 @@
+---
+id: p2p-host-ip
+title: Improve Peer-to-Peer Connectivity
+sidebar_label: Improve P2P connectivity
+---
+
+The ETH2 [architecture](../how-prysm-works/architecture-overview/) is designed to be a fully peer to peer (P2P) network.  This section describes how to configure the Prysm [beacon-chain node](../how-prysm-works/beacon-node) and your network to optimise the number of peers that you communicate with on the ETH2 Network.  Increasing peers helps improve the health, performance and stablity of nodes and the overall network.
+
+> NOTICE: This section contains advanced network configurations and is optional.
+
+ETH2 leverages [libp2p](../how-prysm-works/p2p-networking), a framework and suite of protocols for building peer-to-peer network applications.  When a beacon-chain node first starts up, it does two things to start communicating with other participants: 
+- It begins listening for new incoming P2P connections
+- It starts a [discovery](https://github.com/ethereum/devp2p/wiki/Discovery-Overview) process to find and connect to new peers
+
+![P2P Network Diagram](/img/prysm-p2p-host-ip.png)
+
+## Home networks & routers
+
+Many participants on the ETH2 network operate their nodes on a home network. Home networks typically have a router that provides a logical boundary between your private home network, and the public internet.  While this is good for keeping bad traffic out of your network, it presents a challenge for communicating with other nodes who are also on home networks.
+
+## Virtual public cloud (VPC) networks
+
+Other participants on the ETH2 network operate their nodes on a virtual public cloud (VPC) instance.  This is basically a computer running in a datacenter that quite often is directly connected to the public internet.
+
+## Incoming p2p connection prerequisites
+
+In order for other participants on the ETH2 network to establish incoming p2p connections with your beacon-chain node, a number of conditions must be met:
```

Comment:
P2P still needs to be capitalised throughout (with the exception of libp2p mentions)

Reply:
fixed across the file